pages.ja/common/docker-start.md

```diff
@@ -0,0 +1,20 @@
+# docker start
+
+> 1つまたは複数の停止中コンテナを起動します。
+> 詳しくはこちら: <https://docs.docker.com/engine/reference/commandline/start/>.
+
+- ヘルプを表示する:
+
+`docker start`
+
+- Dockerコンテナを起動する:
+
+`docker start {{コンテナ}}`
+
+- コンテナを起動し、`stdout`(標準出力) と `stderr`(標準エラー出力) をアタッチし、シグナルを転送する:
+
+`docker start --attach {{container}}`
+
+- スペースで区切られた1つまたは複数の停止中コンテナを起動する:
+
+`docker start {{コンテナ1 (コンテナ2)}}`
```

Comment:
I suggest using an ellipsis here.
```suggestion
`docker start {{コンテナ1 コンテナ2 ...}}`
```

Reply:
Sure. Fixed in https://github.com/tldr-pages/tldr/pull/9814/commits/0b3fcc2960948ffd5a7002a763c781ef61824887